src/server/services/schema.js

```diff
@@ -88,6 +89,10 @@ const schema = {
       partners: {
         type: "array",
         validation: validatePartners
+      },
+      partners: {
```

Comment:
I'm not sure this will work with duplicate keys as only one validation function will be called when it does `schema[page]`

Might need to update the custom validation rules (`validator.attributes.validation`) in the validation.service to check for array of validation functions.